src/phoenix/datasets/dataset.py

```diff
@@ -309,6 +320,38 @@ def _check_embedding_features_schema_field_for_excluded_columns(
     )
 
 
+def _check_relationship_schema_field_for_excluded_columns(
+    relationships: Relationships,
+    unseen_excluded_column_names: Set[str],
+    schema_patch: Dict[SchemaFieldName, SchemaFieldValue],
+    column_name_to_include: Dict[str, bool],
+    unseen_column_names: Set[str],
+) -> None:
+    """
+    Check relationships for excluded column names.
+    """
+    included_relationships: Relationships = {}
+    for (
+        relationship_name,
+        relationship_column_name_mapping,
+    ) in relationships.items():
+        included_relationships[relationship_name] = deepcopy(relationship_column_name_mapping)
+        for relationship_field in fields(relationship_column_name_mapping):
+            column_name: Optional[str] = getattr(
+                relationship_column_name_mapping, relationship_field.name
+            )
+            if column_name is not None:
+                column_name_to_include[column_name] = True
+                if column_name in unseen_excluded_column_names:
```

Comment:
nit: someday we should remove this excluded_column business. it's encumbering us with a lot of code for very little value

Reply:
yeah maybe so - I think it's intended to be a compliment to the feature discovery. It actually exists in the arize file importer to reduce the amount of data ingested. From that perspective it might have a place in the long run. Not sure but it def. comes from a customer feature ask where the dataframes are super wide.